docs/image/README.md

```diff
@@ -0,0 +1,16 @@
+# Docs Test
+this container helps to test the documentation before you push to github
+
+## How-to
+
+```
+$ docker build -t local/deis-docs .
+$ docker run --rm -t -i -v `pwd`/../../../deis/deis:/deis -p 8000:8000 local/deis-docs
+```
```

Comment:
Yes, but this requires to setup python, virtualenv, etc. on your system. Having everything already setup in a container a contributor could just download would be awesome.